### ISSUE TITLE
Some portals not functioning

### ISSUE THREAD

**Issue by ouroborus** [TheCyberBrick/The-Betweenlands#588]
#### Installed mods:
ATM3 v5.9.3b https://minecraft.curseforge.com/projects/all-the-mods-3/files/2553021

## Description of the problem
Some portals work, some don't. Ones that don't work refuse to ever work, even if manually re-linked. There is nothing reported in the logs when standing in a non-working portal, the animation just repeats but you don't go anywhere. A new non-working portal doesn't generate an exit portal. It seems like it might be location related. Portals created in the betweenlands, at similar coordinates as a non-working portal in the overworld, will also fail to work.

**Comment by TheCyberBrick** [comment]
Hm strange. Please give me a seed and coordinates for an example where it doesn't work. Can you reproduce this without the ATM3 pack?

And just to be safe, are you sure it's not just some tick lag that prolongs the time needed to stand in the portal to teleport?

**Comment by ouroborus** [comment]
Not sure generating from a seed is going to work very well. ATM3 adds mods often and some of those mods added biomes. This world has gone through several of those updates. I could zip up the world, if you like.

In any case, the seed is "ouroborus" (no quotes) and the coords are 376, 1673.

No, it's not tick lag. Doesn't matter how long you stand there.

**Comment by TheCyberBrick** [comment]
Hmm, I was unable to reproduce this so far with ATM3 v5.9.3b. A world download with a broken portal would help 👍

**Comment by ouroborus** [comment]
[2018-04-25-10-44-53.zip](https://drive.google.com/file/d/1bZowPjRTT1dfVBEGl1J7Hkl805zbEil_) (6.5GB)

**Comment by TheCyberBrick** [comment]
Thanks.
I've tested the portal at 376, 1673 and the one right next to it in the Betweenlands, both seem to work. Also made a bunch of other portals and they all worked.
Is there any other specific portal in the world that doesn't work for you?

**Comment by ouroborus** [comment]
tl;dr: mod interaction, fault is with Overloaded

Knowing that it was just me helped a lot.

After some experimenting, I figured out this is an interaction with the armor from Overloaded. It has a noclip ability. If noclip is on, you can walk through stuff. There's a bug with it though. If you turn on noclip and then change dimensions or relog, the armor will still think you have noclip on, but you can no longer walk through stuff. 

Apparently a side effect with noclip mode is that portals don't work for you. If you just turn on noclip, you don't get an animation and the portal doesn't work. If you turn on noclip and then lose it, portals will animate but still won't transport you.

So somewhere between creating my first portal out in the boonies and creating the portals near my base, I had turned on noclip, changed dimensions, and then forgot about it.
